### PR TITLE
Deprecated in favor of yeoman-app

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # yo-app
 
-**Deprecated**
+**Deprecated in favor of [yeoman-app](https://github.com/yeoman/yeoman-app) which is built on top of Electron**
```
Comment (sindresorhus) [inline readme.md]: I don't see the point in mentioning Electron here. Just:

> Deprecated in favor of yeoman-app

**Comment by sindresorhus** [comment]
https://twitter.com/sindresorhus/status/628569877436084224

**Comment by stevemao** [comment]
Pull request title: "Update readme.md" t(ಠ益ಠt)

**Comment by sindresorhus** [comment]
Yeah, that's not what that meant. Rather, use a proper commit and PR title.

**Comment by stevemao** [comment]
@sindresorhus Done ;)

**Comment by sindresorhus** [comment]
Cool, thanks :)

**Pull request merged**